english/blackAndwhite/blackAndwhite.md

```diff
@@ -16,4 +16,4 @@ wearing a leather hood and jeans, smirks seeing Christy tremble in fear.
 Christy sprints to the window to check if someone is hiding there and sees no one. Stranger drifts into the eternal dark
 and leaves a puzzle behind for her.
 
-Christy's phone vibrates and drops off the table and Christy screams in dismay waking up the entire dorm.
\ No newline at end of file
+Christy's phone vibrates and drops off the table and Christy screams in dismay waking up the entire dorm. Everyone wakes up and walk into her messy room trying to figure out what happened. Christy's best friends Bella and Marie  console her and calm her from trembling.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.